app/src/components/card/Card.tsx

```diff
@@ -0,0 +1,93 @@
+import { forwardRef, Ref, useId, useState } from "react";
+
+import { Heading, Icon, Icons } from "@phoenix/components";
+import { useStyleProps, viewStyleProps } from "@phoenix/components/utils";
+
+import { cardCSS } from "./styles";
+import { CardProps } from "./types";
+
+function Card(
+  {
+    title,
+    subTitle,
+    children,
+    variant = "default",
```

Comment:
I struggle with this variant as well. In general I don't think we have a single "non-compact" card so let's nix it.